defaults/main.yml

```diff
@@ -185,15 +185,16 @@ volume_names:
 
 network_names:
   main: "{{ hub_service_prefix }}_network"
+  internal: "internal_routing"
```

Comment:
to have dev-prod parity, we should also add the network for database servers here.